dataStores/app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const router = require('./router');
+const bodyParser = require('body-parser');
+const app = express();
+
+app.listen(3000, () => console.log('App listening on port 3000!'));
```

Comment:
To się daje na początek, a nie na koniec? Jak zwykle ludzie robią?

Reply:
W sumie nie wiem xd